### ISSUE TITLE
Weird issue with the displaying 2 batteries

### ISSUE THREAD

**Issue by Titaniumtown** [instantOS/instantOS#29]
So have 2 batteries, and where I assume the battery info to be, is all jumbled:
![Screenshot from 2020-06-24 03-20-02](https://user-images.githubusercontent.com/11786225/85514055-9e78eb00-b5c9-11ea-9c12-88ac36f23453.png)

Note: I am not using "instantos" i simply installed the package "instantos" from the repo.

**Comment by NicTanghe** [comment]
how is 98$ and 98$ jumbled ?

**Comment by paperbenni** [comment]
My guess is that they're not actually two, it's just that one is jumping around and doesn't get overdrawn. 
 @Titaniumtown Which locale are you using?

**Comment by Titaniumtown** [comment]
My locale is en_US.UTF-8 what i ment is that there is a weird box in between the numbers (i should have been more clear, sorry) and at least I feel like the 2 numbers should just be simplified to 1 battery percentage.

**Comment by paperbenni** [comment]
Do they always show the same percentage? And is this on your bedrock install?

**Comment by Titaniumtown** [comment]
No, they are correct. but there is a weird box between them. Is that intended? and yes, it is on my bedrock install (but that shouldn't affect anything).

**Comment by shiftgeist** [comment]
![image](https://user-images.githubusercontent.com/30648730/88581039-dc47b400-d04c-11ea-9999-db0383ddcca7.png)

This seems to be the same square, fresh installation on an x240 (2 batteries). Might this have something todo with different keyboard and timezone settings? I have an US keyboard but EU timezone.

**Comment by paperbenni** [comment]
Hm, I don't think it is
The status is getting a complete rework soon though to make it more responsive and reliable

**Comment by elshize** [comment]
Just want to confirm I'm having the same issue, `acpi` prints out two batteries, and I have that weird rectangle between them.

**Comment by ppenguin** [comment]
Probably the output from `acpi` needs to be (optionally?) filtered by the status check. In my case (same issue) `Battery 0` has no info and it would probably work if filtered out.

**Comment by paperbenni** [comment]
@ppenguin  can you post the output that acpi gives on your system?
None of my hardware can create problematic output, but once I have that this should be an easy fix

**Comment by ppenguin** [comment]
> @ppenguin can you post the output that acpi gives on your system?
> None of my hardware can create problematic output, but once I have that this should be an easy fix

Sorry, had completely missed this question, but it bit me again after I installed `nixOS` and `instantWM` on my convertible. (Absolutely blown away by the elegance of `nix`!)
Provided a possible fix in #89 .

To answer the question, my output reads:
```
$ acpi
Battery 0: Unknown, 0%, rate information unavailable
Battery 1: Full, 100%
```
